test/units/plugins/lookup/test_password.py

```diff
@@ -369,14 +379,17 @@ def test_encrypt_no_salt(self):
 
 class TestWritePasswordFile(unittest.TestCase):
     def setUp(self):
-        self.makedirs_safe = password.makedirs_safe
-        self.os_chmod = password.os.chmod
-        password.makedirs_safe = lambda path, mode: None
-        password.os.chmod = lambda path, mode: None
+        self.makedirs_safe = upath.makedirs_safe
+        self.os_chmod = os.chmod
+        os.chmod = lambda path, mode: None
+        self.os_makedirs = os.makedirs
+        os.makedirs = lambda fd, mode: None
+        upath.makedirs_safe = lambda path, mode: None
```

Comment:
You can undo this and in tearDown (and then again below) since the plugin is loaded with `lookup_loader.get('password')` instead of `lookup_loader.get('ansible.builtin.password')`. (I guess the import statement just needed to be updated to match the load name if the fqcn was used)

Reply:
I'll look at doing follow up, really want to simplify a lot of these tests